web/views.py

```diff
@@ -0,0 +1,13 @@
+from django.template import Template, Context
+from django.template.loader import get_template
+from django.http import HttpResponse
+
+from taps.models import Tap
+
+
+def main_page(request):
+    template = get_template('index.html')
+    html = template.render({
+        'tap_count': Tap.objects.count(),
```

Comment:
Should that exclude taps which don't have beers? (i.e. `Tap.objects.filter(beer__isnull=False)`)?